__tests__/app.test.js

```diff
@@ -98,3 +98,42 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status: 200 - responds with an array of comments for the given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toHaveLength(11);
+        expect(body.comments).toBeSortedBy("created_at", {
+          descending: true,
+        });
+        body.comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
+        });
+      });
+  });
+
+  test("status: 404 - responds with 404 not found for an article id that does not exist / article with no comments", () => {
```

Comment:
Nice 404 for article id that does not exist✅
Looking at your test description, should an article with no comments be an error? Has something actually gone wrong with the request, or was it successful and the article doesn't have any comments to get?

Reply:
That makes sense! I have altered it to now return a 200 for an article with no comments :)